src/sciline/typing.py

```diff
@@ -21,19 +11,4 @@
 Graph = dict[Key, Provider]
 
 
-Json = Union[Dict[str, "Json"], List["Json"], str, int, float, bool, None]
-
-
-def get_optional(tp: Key) -> Optional[Any]:
-    if get_origin(tp) != Union:
-        return None
-    args = get_args(tp)
-    if len(args) != 2 or type(None) not in args:
-        return None
-    return args[0] if args[1] == type(None) else args[1]  # noqa: E721
-
-
-def get_union(tp: Key) -> Optional[Tuple[Any, ...]]:
-    if get_origin(tp) != Union:
-        return None
-    return get_args(tp)
```

Comment:
This seems unrelated to the title of this PR. Why did you remove it?

Reply:
It turned out to be partially broken code left over from the change of Optional handling.